core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1413,4 +1413,11 @@ package object config {
     .bytesConf(ByteUnit.BYTE)
     .createOptional
 
+  private[spark] val DEFAULT_CENTRAL_REPOSITORY =
```

Comment:
Move it to `SQLConf`?

Reply:
+1

how about `spark.sql.centralRepository`? `central` doesn't seem to be a useful namespace for configs.

Reply:
Thanks, done in 96b3fe9